engine/src/main/java/org/terasology/logic/characters/interactions/InteractionScreenComponent.java

```diff
@@ -25,5 +25,4 @@
 @API
 public class InteractionScreenComponent implements Component {
     public String screen;
```

Comment:
Guessing this got reverted, if you make further changes might just want to wipe out changes to this class at all :-)